manpages/0.7.x/index.html

```diff
@@ -0,0 +1,57 @@
+<html>
+<head>
+<title>ZFS on Linux - Man pages</title>
+<meta name="keyword" content="zfs, linux, man"/>
+<meta name="description" content="Native port of ZFS to Linux." />
+<meta name="robots" content="all" />
+</head>
+<body>
+<center>
+<a href="index.html"><img title="Native ZFS on Linux" alt="Native ZFS on Linux" src="../../images/zfs-linux.png"></a>
+<br clear="both"><h1>Version 0.7.x</h1>
```

Comment:
Since we're talking about updating these html versions of the man pages as part of the release process I think it makes sense to tie them to a specific version, instead of 0.7.x.  In this case we'd want to start with the 0.7.13 tag.

Reply:
I've relabeled the existing pages as 0.6.5.11 and 0.7.13, but due to the use of some manual work to, uhh, fix up the HTML pages I wouldn't want to do all the 0.6.5.x and 0.7.x versions listed on the homepage. So for now I'm only offering the current versions.

Reply:
That's fine.  Unless we do end up fully automating this, we can just update them for new releases.